sor/src/main/java/com/bazaarvoice/emodb/sor/DataStoreConfiguration.java

```diff
@@ -76,6 +76,19 @@
     @JsonProperty
     private int _deltaBlockSizeInKb = 64;
 
+    @Valid
+    @NotNull
+    @JsonProperty("stashBlackListTableConditionString")
```

Comment:
Minor:  The "String" suffix seems redundant.  Maybe just "stashBlackListTableCondition"?